### DISCUSSION TITLE
Joining multiple pre-trained SpaCy models into one

### DISCUSSION THREAD

**Discussion by aleksandar-devedzic** [explosion/spaCy#7597] [Help: Best practices]
Hello,
Is it possible to join multiple pre trained SpaCy models into one.
For example, I have data on German and English language, but for some reason, when I test German model ("de_dep_news_trf") on English documents, it performs better than English model ("en_core_web_trf") on English documents.

Is is possible to join multiple models, for example:

```
nlp_1 = spacy.load("de_dep_news_trf")
nlp_2 = spacy.load("en_core_web_trf")

nlp = join(nlp_1, nlp_2) # or something like that
```
Is there a way to combine multiple models like this (or in some other way) in order to increase their performance?

**Comment by polm** [comment]
There is no simple way to combine models like that. Theoretically you could make an ensemble but I think you'd need to make a new pipeline component like that.

That said, I am very surprised you find the German model does better on English text than English. What kind of documents are you using it on, and for what task?

**Reply by aleksandar-devedzic** [reply]
Im processing invoices, I have german, english and french docs.
And I have tried models for each language, and german model works the best for all 3 languages
